lib/js/jsbigints.nim

```diff
@@ -0,0 +1,130 @@
+## Arbitrary precision integers.
+## * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt
+when not defined(js) and not defined(nimdoc):
+  {.fatal: "Module jsbigints is designed to be used with the JavaScript backend.".}
+
+type JsBigInt* = ref object of JsRoot ## Arbitrary precision integer for JavaScript target.
+
+func newBigInt*(integer: SomeInteger): JsBigInt {.importjs: "BigInt(#)".}
+  ## Constructor for `JsBigInt`.
+
+func big*(integer: cstring): JsBigInt {.importjs: "BigInt(#)".}
+  ## Constructor for `JsBigInt`.
```

Comment:
show a runnableExample for this proc:
```nim
doAssert big"-1" == big"1" - "big"2"
```
or whatever example you like

Reply:
You can not `Error: invalid indentation.`, theres examples of that below anyways.

Reply:
Use
```nim
func big*(integer: cstring): JsBigInt {.importjs: "BigInt(#)".} =
  ## Constructor for `JsBigInt`.
  runnableExamples: 
   ...
```

Reply:
You can verify them with `nim doc -b:js -r lib/js/jsbigints`